<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>657</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="ip_widget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="portline">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="iplabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="portlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipline">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chat" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>231</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="chat_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="send_widget" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>560</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="send_pushButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
